lib/work_hours_calculator/logger.rb

```
@@ -0,0 +1,54 @@
+        elsif break_start
+          breaks << [break_start, entry[:time]]
+          break_start = nil
+        end
+      end
+
+      {
+        work_start: work_start.strftime("%I:%M:%S %p"),
+        work_end: work_end.strftime("%I:%M:%S %p"),
+        breaks: breaks.map { |start, end_time| [start.strftime("%I:%M:%S %p"), end_time.strftime("%I:%M:%S %p")] }
+      }
+    end
```

RuboCop (1.69.1)

[convention] Assignment Branch Condition size for get_hours_from_log is too high. [<15, 32, 14> 38.01/23]

(Metrics/AbcSize)